scripts/release-utils.js

```diff
@@ -4,6 +4,22 @@ function exec(command) {
   return execSync(command, { encoding: 'utf8' }).trim();
 }
 
+function requireGitHubCLI() {
```

Comment:
If we're going to shell out to `gh`, I thought it would at least be friendly to look before we leap.